core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -48,6 +48,27 @@ private CatalogProperties() {}
 
   public static final boolean CACHE_CASE_SENSITIVE_DEFAULT = true;
 
+  /**
+   * Controls policy of caffeine cache
```

Comment:
I don't think we should express what Cache implementation we used internally. Instead of `caffeine cache` we can say `table cache`